package.json

```diff
@@ -53,6 +54,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-helmet": "^6.1.5",
```

Comment:
What is this used for?

Reply:
ah its something i used for the meta tags thing.

Reply:
I removed it